.github/workflows/uberjar.yml

```diff
@@ -37,7 +39,7 @@ jobs:
 
   check_jar_health:
     runs-on: ubuntu-20.04
-    name: check ${{ matrix.edition }} (java ${{ matrix.java-version }})
+    name: Check if Metabase ${{ matrix.edition }} (with java ${{ matrix.java-version }}) is healthy
```

Comment:
We can't use a name that is too long this, otherwise it will be cut off from the build visualization:

![image](https://user-images.githubusercontent.com/7288/159165986-0d2646ae-fe5c-4cb8-bf09-295d80af69b9.png)

Reply:
Whoops, didn't think that gh had this issue. Fixed